### PR TITLE
updates to doc\MONSTERS.md

### PR DESCRIPTION
**Pull request by Jerimee**
#### Summary
```SUMMARY: None```

#### Purpose of change

```Minor updates to the documentation for name, volume, and weight of monster JSON```

#### Describe alternatives you've considered

```These changes are based upon instructions I received from active/trusted contributors. I double checked and asked follow up questions in an attempt to be as accurate as possible.```

#### Additional context

```only one file should be changed: MONSTERS.md```

### CODE REVIEW

File: doc/MONSTERS.md [hunk 1]
```diff
@@ -24,25 +26,44 @@ Monster types support the following properties (mandatory, except if noted other
 "name": { "ctxt": "fish", "str": "pike", "str_pl": "pikes" }
 ```
 
-Name displayed in-game, and optionally the plural name and a translation context.
+Name displayed in-game, and optionally the plural name and a translation context (ctxt).
```
Comment (anothersimulacrum) [inline doc/MONSTERS.md]: This makes it seem as thought ctxt is mandatory, when it is not, and only needs to be used in ambiguous cases.

Comment (Jerimee) [reply doc/MONSTERS.md]: I agree, but note that the original had the same issue.

Comment (anothersimulacrum) [reply doc/MONSTERS.md]: Actually, no it didn't nor does this one - I missed the optionally bit.

File: doc/MONSTERS.md [hunk 2]
```diff
@@ -24,25 +26,44 @@ Monster types support the following properties (mandatory, except if noted other
...
-## "categories"
-(array of strings, optional)
+## "volume"
+(string)
 
-Monster categories. Can be NULL, CLASSIC (only mobs found in classic zombie movies) or WILDLIFE (natural animals). If they are not CLASSIC or WILDLIFE, they will not spawn in classic mode.  One can add or remove entries in mods via "add:flags" and "remove:flags".
+```JSON
+"volume": "40 L"
+```
+The numeric part of the string must be an integer.
```
Comment (I-am-Erk) [inline doc/MONSTERS.md]: I would add "accepts `l`, `L`, and `ml` as units. Note that `mL` is currently not accepted."

File: doc/MONSTERS.md [hunk 3]
```diff
@@ -24,25 +26,44 @@ Monster types support the following properties (mandatory, except if noted other
...
+
+## "weight"
+(string)
+
+```JSON
+"weight": "3 kg"
+```
+The numeric part of the string must be an integer. Use the largest unit you can keep full precision with. For example: 3 kg, not 3000 g.
```
Comment (I-am-Erk) [inline doc/MONSTERS.md]: I would add "accepts `g` and `kg` as units."

File: doc/MONSTERS.md (hunk 2 above)
Comment (anothersimulacrum) [reply doc/MONSTERS.md]: I don't believe that `l` is accepted.

File: doc/MONSTERS.md (hunk 3 above)
Comment (Jerimee) [reply doc/MONSTERS.md]: This is fine to add this later, but shouldn't hold up this PR. I'll make a note to do that once this PR is complete.

Comment (Jerimee) [reply doc/MONSTERS.md]: I don't know if "resolve conversation" is something that I should do or if @I-am-Erk should do, but I'm going to stab the button in the hopes that it gets this PR merged.

**Comment by Jerimee** [comment]
@I-am-Erk sorry for hassle - I did a bunch of weird stuff to try and clean up github fork and this PR suffers from that. 

This PR should be merged or closed without merging. Let me know if you'd like me to close it myself.

**Comment by I-am-Erk** [comment]
I don't think you need to do final edits in a second PR. Just add the phrases describing what units are accepted to your current changes. It's well within the scope of this PR

**Comment by Jerimee** [comment]
I think I made the changes requested (re units accepted). 

I've made multiple forks (using organizations) and then archived them (including the one associated with this PR) as I flail about making a giant mess of my local repo. I was trying to spare myself the trouble of unarchiving etc etc etc. But I think I did it correctly.

**Pull request merged**